sig-docs/localization-subproject/README.md

```diff
@@ -0,0 +1,104 @@
+# Localization Subproject
+
+The Kubernetes Localization Subproject is owned by [SIG-Docs](https://github.com/kubernetes/community/tree/master/sig-docs) and run by the [Subproject leads](#leadership).
+
+The Kubernetes documentation has been localized into serveral different languages. Each localization is owned by a team of individulas that speaks the language.  A localization team is a selfsufficent community responsible for prioritizing work, translating content, and reviewing and merging PRs into the kubernetes/website repo for their localization. 
+
+The Localization Subproject's aim is to provide a place for localization teams to share ideas and standardize processes across all localizations. This page covers documentation, processes, and roles for the Kubernetes Localization teams and the Localization Subproject.
+
+## Meetings
+
+Regular Localization Meeting: The first Monday of the month at 15:00 UTC (biweekly). [Convert Your Timezone](http://www.thetimezoneconverter.com/?t=15:00&tz=UTC)
+
+- [Meeting notes and Agenda](https://docs.google.com/document/d/1NwO1AN8Ea2zlK8uAdaDAKf1-LZDAFvSewIfrKqfl5No/edit#)
+- [Meeting recordings](https://www.youtube.com/playlist?list=PL69nYSiGNLP3b5hlx0YV7Lo7DtckM84y8)
+
+A localization team may also hold language specific meetings. If you are interested in attending a language specific meeting see the [Localization Teams](#localization-teams) section for details on how to contact each localization team.
+
+## Subproject Leadership
+
+- **Subproject lead:** [Abigail McCarthy](https://github.com/a-mccarthy)
+
+- **Emeritus leads:** [Brad Topol](https://github.com/bradtopol)
+
+## Contact
+
+- Slack: [#sig-docs-localizations](https://kubernetes.slack.com/archives/C0191RDKHU1) Each localiztion also has a language specific channel, noted [below](#localization-teams). 
+- Mailing list: 
```

Comment:
We have this, but I don't think there is a mandate for the localization leads/contributors to join this group. I see very few members (~20) in the group below. A related topic for broader discussion is how we should be aligning the localization meeting & language-specific meeting invites to this google group so that we can ensure streamlining of comms. For sure, the former one has been created under SIG Docs official. 

Thoughts?

```suggestion
- Mailing list: https://groups.google.com/g/kubernetes-sig-docs-localization
```

Reply:
This is a good point @divya-mohan0209! We have the mailing list, but we don't use it too much, except for meeting reminders. And we don't require people to join it. 

I'm really not sure if we have the need for it, but maybe we can ask for feedback to see if folks would prefer to get communication in email or slack. But if we decide to use the localization mailing list, we should make sure folks join it. 

The localization meeting invite is through the main sig docs mailing list, not the localization mailing list.